src/calculator.js

```diff
@@ -0,0 +1,84 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+
+const Button = (props) => {
+  const { label } = props;
+  return (
+    <button type="button">{label}</button>
+  );
+};
+
+const Screen = () => (
+  <div className="screen-row">
+    <input type="text" value={0} readOnly />
+  </div>
+);
```

Comment:
- [ ] [optional] I would suggest moving these components to a separate files, It is a good practice to keep components in separate files.